tests/versioned/apollo-federation/package.json

```diff
@@ -27,7 +27,8 @@
         "apollo-server": "2.25.1"
       },
       "files": [
-        "segments.test.js"
+        "segments.test.js",
```

Comment:
so i added 2 new tests but forgot to add as well:
 * https://github.com/newrelic/newrelic-node-apollo-server-plugin/pull/100/files
 * https://github.com/newrelic/newrelic-node-apollo-server-plugin/pull/97/files

also there are 2 diff stanzas here that the test must be added to

Reply:
The other missing versioned tests were updated with this PR https://github.com/newrelic/newrelic-node-apollo-server-plugin/pull/113